autounsuspend.lua

```diff
@@ -0,0 +1,34 @@
+-- automate periodic running of the unsuspend script
+--[====[
+
+autounsuspend
+=============
+Periodically check construction jobs and keep them unsuspended with the
+`unsuspend` script.
+]====]
+
+local job_name = '__autounsuspend'
+
+local function help()
+    print('syntax: autounsuspend [start|stop]')
+end
+
+local function stop()
+    dfhack.run_command{'repeat', '-cancel', job_name}
+    print('autounsuspend Stopped.')
+end
+
+local function start()
+    dfhack.run_command{'repeat', '-name', job_name, '-time', '1',
```

Comment:
I would actually suggest using the `repeat-util` module for this (which is what repeat uses) - it would avoid the need to construct these commands by hand.

And as a side note, `run_script()` has somewhat less overhead than `run_command()` (but only works for Lua scripts)

Reply:
done. thanks!